.github/workflows/build.yml

```diff
@@ -2,25 +2,30 @@ name: Build oee-simulators
 
 on:
   workflow_call:
+    inputs:
+      release_body:
+        type: string
+        required: true
 
 jobs:
   build-upload:
     runs-on: ubuntu-latest
+    outputs:
+      release_version: ${{ env.release_version }}
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v3
         with:
           path: 'repo'
 
-      - name: Set branch name
-        run: echo "BRANCH_NAME=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
-
       - name: Update version number
         run: |
           cd repo/simulators
           SIMULATOR_VERSION=${GITHUB_RUN_NUMBER}
           sed -i "s/@project.version@/1.0.${SIMULATOR_VERSION}/g" cumulocity.json
-          echo "RELEASE_VERSION=${SIMULATOR_VERSION}" >> $GITHUB_ENV
+          echo "release_version=${SIMULATOR_VERSION}" >> $GITHUB_ENV
+        env:
+          release_version: ${{ env.release_version }}
```

Comment:
is that really needed? I dont see the value of `env.release_version` being used in the step ..

Reply:
Yes, it is passed to the release workflow